avaliacao360/interface/evaluation.py

```diff
@@ -26,19 +26,26 @@ def event_handler(event, values):
         nonlocal evaluation, evaluated_index
         if event == 'Enviar':
             result = []
-            #para cada tela capturar a combinacao de resultados
-            #iterar um contador para nomear o botao de radio capturado
+            
             def callback(group_id, _):
                 if values['1_' + group_id]:
-                    result.append(1)
+                    feedback = sg.popup_get_text(
+                        "Justifique sua resposta: ",
+                        title=question_list[int(group_id[-1])]
+                        )
+                    result.append({'value':1, 'feedback':feedback})
                 if values['2_' + group_id]:
-                    result.append(2)
+                    feedback = sg.popup_get_text(
+                        "Justifique sua resposta: ",
+                        title=question_list[int(group_id[-1])]
+                        )
+                    result.append({'value':2, 'feedback':feedback})

```

Comment:
```suggestion
                    feedback = sg.popup_get_text(
                        "Justifique sua resposta: ",
                        title=question_list[int(group_id[-1])]
                        )
                    result.append({'value':2, 'feedback':feedback})
```
Mesma coisa que o do `if values['1_' + group_id]`

Reply:
Resolvido no [1ffdf70](https://github.com/wizards-of-api/avaliacao-360/pull/27/commits/1ffdf709dc1fb13ef65a3d43204afff5b72dc813) e [38fbca8](https://github.com/wizards-of-api/avaliacao-360/pull/27/commits/38fbca875c7717c1b827a1bfead26927c08c07a3)